<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjustFontsDialog</class>
 <widget class="QDialog" name="AdjustFontsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font Size Adjustment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QListWidget" name="listTextItems"/>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QLabel" name="lblFontAdjustStatus">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="btnFontAdjustRunProcess">
     <property name="text">
      <string>Start</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnToggleCheckAll">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="select-all">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cmbFontAdjustOperator">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Adding</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dividing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multiplying</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Subtracting</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Setting exactly to</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="floatAdjustFontAmount">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.000001000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.750000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Adjust font size by:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdjustFontsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdjustFontsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
